styles.css

```diff
@@ -0,0 +1,101 @@
+
+/* CSS stylesheet */
+.mainheader {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.toolbar {
+  position: absolute;
+  width: 1440px;
+  height: 72px;
+  left: 0;
+  top: 0;
+}

```

Comment:
- [ ] Kindly follow the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) and use CSS flexbox to style the toolbar (header).
It's so important to practice using flexbox and apply some cool tricks with it. [This](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) might help you!